scripts/download_artifacts.py

```diff
@@ -294,57 +397,19 @@ def download_all_artifacts(
     # TODO: Refactor this block
```

Comment:
```suggestion
```

Reply:
or move to where the code has been copied to if you think it could use further work.

Reply:
This block was supposed to include the entire for loop and I am not sure what the intention was behind this comment. It claims that it was added by you 9 months ago. Do you have any suggestions for the new location?

Reply:
I don't remember the original intent - if the code quality for the for loop contents seems to be on-par with the rest of the file then I'd drop the comment.